HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc

```diff
@@ -273,21 +273,41 @@ void HLTScoutingEgammaProducer::produce(edm::StreamID sid, edm::Event& iEvent, e
         continue;
     }
 
-    float d0 = 0.0;
-    float dz = 0.0;
-    int charge = -999;
+    bool tracknotfound = true;
+    unsigned int trksize = EgammaGsfTrackCollection->size();
```

Comment:
```suggestion
    unsigned int const trksize = EgammaGsfTrackCollection->size();
```
(maybe call it `maxTrkSize`)